css/contact.css

```diff
@@ -0,0 +1,225 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;600&display=swap');
+
+
+
+:root{
+    --primary:#0150FD;
```

Comment:
Colors in CSS vars as spec requested, nice, it did ask for fonts to be in vars also though